CMakeLists.txt

```diff
@@ -18,6 +18,49 @@ option(ENABLE_DEVEL "Enable development asserts, checks, etc" OFF)
 option(ENABLE_REFCNT_DEBUG "Enable refcnt debugging" OFF)
 option(ENABLE_SHAREDPTR_DEBUG "Enable sharedptr debugging" OFF)
 set(TRYCOMPILE_SRC_DIR "${CMAKE_CURRENT_LIST_DIR}/packaging/cmake/try_compile")
+set(BUILT_WITH "")
+
+# Toolchain {
+if(CMAKE_C_COMPILER_ID STREQUAL "GNU")
+  list(APPEND BUILT_WITH "GCC")
+elseif(CMAKE_C_COMPILER_ID STREQUAL "Clang")
+  list(APPEND BUILT_WITH "CLANG")
+elseif(CMAKE_C_COMPILER_ID STREQUAL "MSVC")
+  list(APPEND BUILT_WITH "MSVC")
+endif()
+
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  list(APPEND BUILT_WITH "GXX")
+elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+  list(APPEND BUILT_WITH "CLANGXX")
+elseif(CMAKE_C_COMPILER_ID STREQUAL "MSVC")
+  list(APPEND BUILT_WITH "MSVCXX")
+endif()
+# }
+
+# GNU install {
+find_program(GnuInstall "install")
```

Comment:
Not sure if this is needed with cmake, it already does the install thing, right?
And as for build option, INSTALL isn't relevant, but everything is included in the mklove string.

Reply:
I think you're right that CMake is not using GNU install for the install procedure. I will remove it.

Reply:
Go for it!

Reply:
Removed in https://github.com/edenhill/librdkafka/commit/b773fa76629c0ac6142a854720904a5a1be8556d.